### DISCUSSION TITLE
Possibility to exclude some letter combinations

### DISCUSSION THREAD

**Discussion by knstntnb** [shlinkio/discussions#100] [Help wanted] [answered]
Hi,
I finally got Shlink up & running - works like expected and after finalizing the tests, I want to switch from my yourls-install to Shlink. :)

With all the testing, I just noticed that I got some letter combinations, which I want to prevent in the future (e.g. "FCK3v" or "PsY1l". I read that there is only a docker env about the URL-length, but nothing about the matter above.

Is there such a possibility right now, or planned? 
The workflow I am using is via an app on my Mac or mobile devices with auto-shortening.

**Comment by acelaya** [comment, marked as answer]
Not supported at the moment, or planned.

In the very early days of Shlink, it was possible to customize the charset, but it usually brought little vale (because usually people don't care), and introduced confusion, so I decided to remove it long ago.

This is a valid concern though. I know some libraries which generate short codes/IDs and make sure certain characters are never next to each other to avoid this.

Feel free to open a feature request and we can explore the options.

**Reply by knstntnb** [reply]
Thanks for replying and answering - will open a feature request for this! :)
